wcfsetup/install/files/lib/data/package/update/server/PackageUpdateServer.class.php

```diff
@@ -52,6 +52,13 @@ protected function handleData($data) {
 		}
 		
 		parent::handleData($data);
+		
+		if ($this->isWoltLabUpdateServer()) {
+			$this->data['serverURL'] = 'http://update.woltlab.com/5.3/';
```

Comment:
Using (the major and minor version of) `WCF_VERSION` would avoid having the version number hard-coded.